protokt-codegen/src/main/resources/renderers.stg

```diff
@@ -129,6 +129,23 @@ isScalar ::= [
     default: false
 ]
 
+javaTypeNames ::= [
```

Comment:
I thought this was moving to the options module

Reply:
It's headed there (in #19).